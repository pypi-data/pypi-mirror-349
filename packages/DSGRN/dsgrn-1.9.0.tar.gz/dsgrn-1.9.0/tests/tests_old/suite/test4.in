#!/bin/bash
$1 network ../networks/4D_Example.txt morsegraph graphviz '[["X",[3,3,"FF8EC8"],[2,0,1]],["Y1",[1,1,"2"],[0]],["Y2",[1,1,"2"],[0]],["Z",[1,1,"2"],[0]]]'
