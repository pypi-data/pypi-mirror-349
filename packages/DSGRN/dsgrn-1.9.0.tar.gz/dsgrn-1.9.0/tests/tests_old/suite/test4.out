digraph {
0[label="PC {X, Z}"];
1[label="PC {X, Z}"];
2[label="PC {X, Z}"];
3[label="PC {X, Z}"];
4[label="PC {X, Y1, Y2}"];
5[label="FP { 2, 0, 0, 1 }"];
6[label="FP { 3, 1, 1, 1 }"];
0 -> 1;
1 -> 2;
2 -> 3;
2 -> 4;
4 -> 5;
4 -> 6;
}
